build.sbt

```diff
@@ -80,6 +79,7 @@ lazy val root = (project in file("."))
 lazy val common = (project in file("common"))
   .settings(
     name := "common",
+    scalacOptions := compilerFlags,
```

Comment:
this was missing by accident, adding it flagged up a few more deprecation warnings